handler/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("@rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "handler",
```

Comment:
I couldn't find anything about gazelle generating a build rule per file. It seems to always base it on the whole package.

That being said, whats the issue with using the same build rule for all of the handlers? I imagine the dependencies will be required for most of them. We could split up the handlers into different categories to split up the build rules if needed

e.g all CRUD-like handlers go into a folder handler/crud/create_user.go

Reply:
when a build target fails you would know exactly which file is impacted, so narrowing in on debugging would be easier. when you make a change, the build target has to be rebuilt, if there is one file per build target then you minimize the amount of rebuilding you do. it is also used for fine-grained visibility control so you know at the build-level which files depend on what. when you group a bunch of files together, then you are uncertain which file needs or provides on which other file. easier to track unused dependencies as well, since you only have to update the deps for the single file of the target. up to you ser

Reply:
oh also forgot to mention when you are testing, and a target fails, you know exactly which file. and when refactoring it is much simpler and error-resistant to delete a target

Reply:
Welp I guess we're creating a directory per file then rip